config/checkstyle_resources_suppressions.xml

```diff
@@ -175,6 +175,12 @@
   <suppress checks="PackageDeclarationCheck"
             files="[\\/]InputAstTreeStringPrinterTextBlocksEscapesAreOneChar\.java"/>
 
+  <!-- AST regression testing requires complicated input, we should not limit input file length -->
```

Comment:
I will probably send this as a separate PR to speed things up, just keeping CI green now.